### PR TITLE
chore: bump frontify packages

### PR DESCRIPTION
**Pull request by fulopdaniel**

### CODE REVIEW

File: examples/asset-upload/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     },
     "dependencies": {
         "@frontify/app-bridge": "^3.3.1",
-        "@frontify/fondue": "12.0.0-beta.404",
+        "@frontify/fondue": "12.0.0-beta.406",
         "@frontify/guideline-blocks-settings": "^0.33.1",
```
Comment (ragi96) [inline examples/asset-upload/package.json:36]:
```suggestion
        "@frontify/guideline-blocks-settings": "^0.34.0",
```

**Review by ragi96 — COMMENTED** [review]
Can you also check that everywhere `import '@frontify/fondue/style';` is added so all blocks are fine? Its missing on the most:
Screenshot 2024-02-26 at 11 45 53

File: examples/asset-upload/package.json (hunk 1 above)
Comment (fulopdaniel) [reply examples/asset-upload/package.json]: Thanks fixed. And added fondue styles to all blocks.

File: packages/shared/package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
     "dependencies": {
         "@codemirror/lang-css": "^6.2.1",
         "@frontify/app-bridge": "^3.3.1",
-        "@frontify/fondue": "12.0.0-beta.404",
+        "@frontify/fondue": "12.0.0-beta.406",
         "@frontify/guideline-blocks-settings": "0.33.2",
```
Comment (ragi96) [inline packages/shared/package.json:47]:
```suggestion
        "@frontify/guideline-blocks-settings": "^0.34.0",
```

Comment (fulopdaniel) [reply packages/shared/package.json]: oops, added

**Review by ragi96 — APPROVED** [review]

**Pull request merged**